server/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+const db = require('./models');
+
+app.use(express.json());
+
+const PORT = process.env.PORT || 3000;
+
+
+// Sync Sequelize models and create a test user
+db.sequelize.sync({ force: true }).then(async () => { // Use { force: true } cautiously as it will drop existing tables
+    console.log('Database synced');
+
+    // Create a test user
+    try {
```

Comment:
i believe this is for testing purposes, right?

If yes, please mark it as such.

Reply:
Exactly. 
Done